### PR TITLE
implement number of user choosing the same choice

### PR DESCRIPTION
**Pull request by vinhyenvodoi98**
implement

show number of user choosing the same choice

### CODE REVIEW

File: src/component/PlayerLayout/PlayerLayout.js [hunk 1]
```diff
@@ -44,24 +44,24 @@ class PlayerLayout extends Component {
         <Container>
           {tomo.web3 ? (
             question ? (
-              <Row className='set_height'>
+              <Row className="set_height">
                 <QuesArea ques={question} acc={this.props.tomo} />
```
Comment (tranchien2002) [inline src/component/PlayerLayout/PlayerLayout.js]: " -> '

**Comment by vinhyenvodoi98** [comment]
```javascript
// Total calculation function 
sumValues = (obj) => Object.values(obj).reduce((a, b) => a + b);

if (this.sumValues(this.state.user_choice) !== 0) {
   return (user_number * 100) / this.sumValues(this.state.user_choice);
} else {
   return 0;
}
```

File: src/component/PlayerLayout/PlayerLayout.js (hunk 1 above)
Comment (vinhyenvodoi98) [reply src/component/PlayerLayout/PlayerLayout.js]: (done)

**Comment by vinhyenvodoi98** [comment]
image

**Pull request merged**